frame/offences/src/tests.rs

```diff
@@ -71,14 +75,34 @@ fn should_not_report_the_same_authority_twice_in_the_same_slot() {
 	});
 }
 
+#[test]
+fn should_not_report_an_obsolete_offence() {
+	new_test_ext().execute_with(|| {
+		// given
+		let time_slot = 42;
```

Comment:
what is the unit/meaning of time_slot?

Reply:
Just a slot within a session? Could you please explain the question a bit more?